docs/docs/guides/net_migration_guide.md

```diff
@@ -1,3 +1,8 @@
+---
+sidebar_position: 6
```

Comment:
there are already [7](https://github.com/ChainSafe/web3.js/blob/4.x/docs/docs/guides/personal_migration_guide.md) pages so it should be at 8 ?